src/changelog/2024/12/device-editor-cache.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Device Editor Access Speed Up
+description:
+date: 2024-12-18 14:00:00.0
+authors: ["ben-hardill"]
+tags:
+    - changelog
+---
+
+With the 2.12 release we have included some improvements to the Device Editor experience.
+
+For Devices running with Node-RED 4.0.x and newer, there should be performance improvements when loading the editor, especially when the device is on a slow network link.
```

Comment:
```suggestion
For Devices running with Node-RED 4.0.x and newer, there should be significant performance improvements when loading the editor, especially when the device is on a slow network link.
```

Suggestion only - lets big this up a _little_ - its a great improvement worthy of shouting about ;)